### ISSUE TITLE
error[E0554]: #![feature] may not be used on the stable release channel

### ISSUE THREAD

**Issue by arden** [termhn/nano-rs#2]
cargo run --release

```
error[E0554]: #![feature] may not be used on the stable release channel
 --> nano-lib-rs/src/lib.rs:2:1
  |
2 | #![feature(i128_type)]
  | ^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

error: Could not compile `nano-lib-rs`.

To learn more, run the command again with --verbose.
```

**Comment by fu5ha** [comment]
Ah yeah should add this to docs but you need to be on nightly Rust which
you can do by

```
rustup toolchain install nightly
rustup default nightly
```
(Or various other ways, this will set nightly as your default tool chain)

**Comment by fu5ha** [comment]
Fixed in commit 9bb7841
